js/components/siteInfo.js

```diff
@@ -204,6 +206,7 @@ class SiteInfo extends ImmutableComponent {
       connectionInfo =
         <div className={css(styles.connectionInfo__wrapper)}>
           <div data-l10n-id='insecureConnectionInfo' />
+          {viewCertificateButton}
```

Comment:
i think this should only appear if the site is actually HTTPS, otherwise clicking the button doesn't do anything.

![screen shot 2017-04-27 at 22 50 19](https://cloud.githubusercontent.com/assets/549654/25507508/0260cee6-2b9c-11e7-9742-82c31e73e205.png)